components/governance/org.wso2.carbon.governance.registry.extensions/src/org/wso2/carbon/governance/registry/extensions/utils/CommonUtil.java

```diff
@@ -275,6 +278,13 @@ public boolean accept(File dir, String name) {
         }
     }
 
+    private static void updateTenantsUnboundedFieldMap(GovernanceArtifactConfiguration rxtConfiguration)
+            throws RegistryException {
+        if (rxtConfiguration.getMediaType().matches("application/vnd.(.)+\\+xml")) {
+            RXTIndexer.updateTenantsUnboundedFieldMap(rxtConfiguration.getContentDefinition().getParent().toString());
```

Comment:
Shall we implement the logic here instead referring from RXTIndexer. It would be better if we can have the logic in utils class refer it in indexer.
We can make this public and accept tenant Id as well.
WDYT?